compiler/passes/buildDefaultFunctions.cpp

```diff
@@ -1475,18 +1497,7 @@ static void buildDefaultReadWriteFunctions(AggregateType* ct) {
                                     fn->_this));
     }
 
-    DefExpr* def = new DefExpr(fn);
-
-    ct->symbol->defPoint->insertBefore(def);
-
-    fn->setMethod(true);
-    fn->addFlag(FLAG_METHOD_PRIMARY);
-
-    reset_ast_loc(def, ct->symbol);
-
     normalize(fn);
```

Comment:
Does the ordering of `normalize` not matter here?

Reply:
I think that's right.  Logically it seemed like it shouldn't matter (*), and testing seems to have confirmed that.

(* = normalize should only modify the definition of the function, and shouldn't matter whether it's inserted into the AST already or not).